doc/src/guides/solving/solve-ode.md

```diff
@@ -0,0 +1,467 @@
+example, the value of `f(x)` at $x = $ `x0` is `y0`. For power series solutions,
+if no initial conditions are specified `f(0)` is assumed to be `C0` and the
+power series solution is calculated about $0$.
+
+Here is an example of setting the initial values for functions, namely namely
+$f(0) = 1$ and $g(2) = 3$:
+
+```py
+>>> from sympy import symbols, Eq, Function, dsolve
+>>> f, g = symbols("f g", cls=Function)
+>>> x = symbols("x")
+>>> eqs = [Eq(f(x).diff(x), g(x)), Eq(g(x).diff(x), f(x))]
+>>> dsolve(eqs, [f(x), g(x)])
+[Eq(f(x), -C1*exp(-x) + C2*exp(x)), Eq(g(x), C1*exp(-x) + C2*exp(x))]
+>>> dsolve(eqs, [f(x), g(x)], ics={f(0): 1, g(2): 3})
+[Eq(f(x), (1 + 3*exp(2))*exp(x)/(1 + exp(4)) - (-exp(4) + 3*exp(2))*exp(-x)/(1 + exp(4))), Eq(g(x), (1 + 3*exp(2))*exp(x)/(1 + exp(4)) + (-exp(4) + 3*exp(2))*exp(-x)/(1 + exp(4)))]
+```
+
+Here is an example of setting the initial value for the derivative of a
+function, namely $f'(1) = 2$:
+
+```py
+>>> dsolve(eqs, [f(x), g(x)], ics={f(x).diff(x).subs(x, 1): 2})
+[Eq(f(x), C2*exp(x) + (C2*exp(2) - 2*E)*exp(-x)), Eq(g(x), C2*exp(x) - (C2*exp(2) - 2*E)*exp(-x))]
+```
+
+## Define a Function of an Independent Variable
+
+You may prefer to specify a function (for example $x$) of its independent
+variable (for example $t$), so that `x`, represents `x(t)`:
+
+```py
+>>> from sympy import symbols, Function, dsolve
+>>> t = symbols('t')
+>>> x = Function('x')(t)
+>>> x
+x(t)
+>>> xp = x.diff()
+>>> xpp = xp.diff()
+>>> eq = xpp + 2*xp + x
+>>> eq
+x(t) + 2*Derivative(x(t), t) + Derivative(x(t), (t, 2))
+>>> dsolve(eq, x)
+Eq(x(t), (C1 + C2*t)*exp(-t))
+```
+
+Using this convention, the second argument of {func}`~.dsolve`, `x`, represents
+`x(t)`, so SymPy recognizes it as a valid function to solve for.
+
+### Specify Initial (Boundary) Conditions
+
+Using that syntax, you specify initial conditions by substituting in values of
+the independent variable using {func}`~sympy.core.basic.Basic.subs` because the
+function $x$ already has its independent variable as an argument $t$:
+
+```py
+>>> dsolve(eq, x, ics={x.subs(t, 0): 0})
+Eq(x(t), C2*t*exp(-t))
+```
+
+### Beware Copying and Pasting Results
+
+If you choose to define a function of an independent variable, note that copying
+a result and pasting it into subsequent code may cause an error because `x` is
+already defined as `x(t)`, so if you paste in `x(t)` it is interpreted as
+`x(t)(t)`:
+
+```py
+>>> dsolve(x(t).diff(), x)
+Traceback (most recent call last):
+    ...
+TypeError: 'x' object is not callable
+```
+
+So remember to exclude the independent variable call `(t)`:
+
+```py
+>>> dsolve(x.diff(), x)
+Eq(x(t), C1)
+```
+
+## Use the Solution Result
+
+Unlike other solving functions, {func}`~.dsolve` returns an {class}`~.Equality`
+(equation) formatted as, for example, `Eq(y(x), C1*sin(3*x) + C2*cos(3*x))`
+which is equivalent to the mathematical notation $y(x) = C1 \sin(3x) + C2
```

Comment:
Make sure to subscript the constants in LaTeX (`C_1`). You can also generally just use `print(latex(expr))` on an expression to get a LaTeX form to put in the docs.